### PR TITLE
Onboard for Approval decision utils

### PR DESCRIPTION
**Pull request by nntthuy-axonivy**

### CODE REVIEW

**Comment by nntthuy-axonivy** [comment]
Hi @andreasbalsiger, I created this pull request for the documentation step.
It describes the purposes and functionality of the Approval decision utils and will be shown on the axon ivy market landing page.
Thanks!

File: market/utils/approval-decision-utils/logo.png
Comment (andreasbalsiger) [inline market/utils/approval-decision-utils/logo.png:1]: The icon is not suitable. It is old and overused. try to take this one:
logo

Comment (nntthuy-axonivy) [reply market/utils/approval-decision-utils/logo.png:1]: Hi @andreasbalsiger, thanks for your recommendation. I have committed a new logo for this utils. Please have a look on it!

File: market/utils/approval-decision-utils/meta.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://json-schema.axonivy.com/market/10.0.0/meta.json",
+	"id": "approval-decision-utils",
+	"name": "Approval Decision Utils",
+	"description": "Empower applications with Approval Decision: Accelerate decision-making and streamline processes",
```
Comment (andreasbalsiger) [inline market/utils/approval-decision-utils/meta.json:5]: Standardize decision-making in business processes to meet regulatory, compliance, and transparency requirements.

Comment (nntthuy-axonivy) [reply market/utils/approval-decision-utils/meta.json]: hi @andreasbalsiger, I've modified the artifact description. Kindly review the PR, thank you!

**Pull request merged**